modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/manager/DefaultExtensionManager.java

```diff
@@ -81,7 +82,7 @@
   private ExtensionActivator extensionActivator;
 
   @Override
-  public void initialise() throws InitialisationException {
+  public void initialise() {
```

Comment:
You are throwing exceptions, make them throw `InitialisationException` if not, you are not compliant with the API.